src/libs/README.md

```diff
@@ -1,3 +1,10 @@
 # Libs
-This section documents meta processes by the libs team.
 
+This section documents meta processes by the Libs team.
+
+## Where to find us
+
+The libs team hangs out primarily in [the rust-lang Zulip](https://rust-lang.zulipchat.com/) these days in the `#t-libs` stream.
+You can find more details about Zulip and how the Rust community uses it [here](../../chat/zulip).
```

Comment:
The URL is slightly off, and I changed the language slightly to avoid "click here" as that is considered bad style in my experience.
```suggestion
You can also find out more details about [Zulip and how the Rust community uses it](/chat/zulip.html).
```